t/gcp_project_services/environment-terraform.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    google = {
+      version = ">= 4.45"
```

Comment:
I wonder if this should be an abc cli variable with a default value.

Reply:
Im leaning towards having the template owner manage this. Because they are the ones building the module and having expectations around what is "required".